numpy/core/numerictypes.py

```diff
@@ -124,7 +124,7 @@
 generic = allTypes['generic']
 
 genericTypeRank = ['bool', 'int8', 'uint8', 'int16', 'uint16',
-                   'int32', 'uint32', 'int64', 'uint64', 'int128',
+                   'int32', 'int64', 'int128',
```

Comment:
```suggestion
                  'int32', 'uint32', 'int64', 'uint64', 'int128',
```
Sorry, I had forgotten about this, after seeing this, pausing and not coming back.

I will just revert this, unless you had some reason?  This change seems unrelated to the rest, and it is a bit worrying that no test notices the change.  (But since it is unrelated, lets ignore that.)

Reply:
you can revert